README.md

```diff
@@ -18,11 +18,57 @@ In your `Podfile`, add the following entry:
 pod 'ChartboostMediationAdapterReference'
 ```
 
-## Chartboost Mediation Custom Adapter Implementation Guidelines
+## Chartboost Mediation Custom Adapter Implementation Guide
 
 > [!IMPORTANT]
 > Chartboost Mediation will not be providing official support for any custom adapters. For official adapters that we develop and support, [visit this site](https://adapters.chartboost.com).
 
+1. Create a new class and conform to Helium’s `PartnerAdapter` protocol.
+2. Implement var `partnerSDKVersion: String { get }` with the version of the partner SDK. Most adapters derive this from the partner SDKs API so that it always reports the correct number, even when a newer version is imported.
+3. Implement `var adapterVersion: String { get }` with the version of the mediation adapter. The adapter version format is `<Chartboost Mediation major version>.<Partner major version>.<Partner minor version>.<Partner patch version>.<Partner build version>.<Adapter build version>`.  
+`<Partner build version>` is optional, and omitted by most partners."
+For example, if this adapter is compatible with Helium SDK 4.x and partner SDK 1.2.3.x, and this is its initial release, then adapterVersion is 4.1.2.3.x.0.
+4. Implement var `partnerIdentifier: String { get }` with the internal identifier that the Helium SDK can use to refer to the current partner. Must match the value used in Chartboost Mediation's backend.
```

Comment:
```suggestion
4. Implement `var partnerIdentifier: String { get }` with the internal identifier that the Chartboost Mediation SDK can use to refer to the current partner. Must match the value used on the Chartboost Mediation dashboard.
```

I think the custom adapter identifier will be input by the publisher on the dashboard, but we should confirm this.